doc/source/math_num_documentation/forward_structure.rst

```diff
@@ -351,8 +351,120 @@ Hydrological processes can be described at pixel scale in `smash` with one of th
     Same as ``gr4`` transfer, see :ref:`GR4 Transfer <math_num_documentation.forward_structure.hydrological_module.gr4>`
 
 
-.. _math_num_documentation.forward_structure.hydrological_module.gr6:
+.. _math_num_documentation.forward_structure.hydrological_module.gr5_ri:
+
+.. dropdown:: gr5_ri (Génie Rural 5 with rainfall intensity terms)
+    :animate: fade-in-slide-down
+
+    This hydrological module is derived from the model introduced in :cite:p:`Astagneau_2022`.
+
+        
+    **Production**
+    
+
+    In the classical gr production reservoir formulation, the instantaneous production rate is the ratio between the state and the capacity of the reservoir :
+    :math:`\eta = \left( \frac{h_p}{c_p} \right)^2`. The integration of the complementary of :math:`\eta` the rainfall infiltration :math:`p_s`.
```

Comment:
Tu vas peut-être reformuler cette phrase (elle n'est pas hyper claire pour moi) : "The integration of the complementary of :math:`\eta` the rainfall infiltration :math:`p_s`."

Reply:
oui reformuler par ex.: The infiltration flux is obtained by temporal integration as follows: